File: templ.py
from pathlib import Path
from itertools import *
from collections import *
from functools import *
from operator import *
from heapq import *
from math import *
from statistics import median, mean
import re
import numpy as np
import time

paths = [Path(__file__).with_name('sample.txt'), Path(__file__).with_name('input.txt')]

def solveA(data):
    return

def solveB(data):
    return

def main():
    for p in paths:
        with p.open('r') as f:
            data = f.read()
            print(data)
            print(solveA(data))
            print(solveB(data))

if __name__ == '__main__':
    main()

File: templ.cpp
#include <bits/stdc++.h>

int main() {
    for (auto file : {"sample.txt", "input.txt"}) {
        std::ifstream input {file};
        if (!input) {
            std::cerr << "Failed to open: " << file << '\n';
            return 1;
        }
        std::string line;
        while (std::getline(input, line)) {
            std::cout << line << '\n';
        }
        std::cout << file << ":\n";
        std::cout << "Answer part 1:  " << "..." << '\n';
        std::cout << "Answer part 2:  " << "..." << '\n';
    }
    return 0;
}
